### PR TITLE
[production] Change release of packages to experimental

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: packages/system/0.5.2/manifest.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ type: integration
 categories:
 - os_system
 - security
-release: beta
+release: experimental
```
Comment (ruflin) [inline packages/system/0.5.2/manifest.yml:11]: @exekias @andresrc @fearful-symmetry This is for 7.10 what we have today. Is this change ok? I assume we will release a new version in any case for 7.10?

File: packages/endpoint/0.13.0/manifest.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ description: Protect your hosts with threat prevention, detection, and deep secu
 version: 0.13.0
 categories: ["security"]
 # Options are experimental, beta, ga
-release: beta
+release: experimental
```
Comment (ruflin) [inline packages/endpoint/0.13.0/manifest.yml:8]: @jonathan-buttner Ups, find replace. I assume this should stay beta, correct?

Comment (jonathan-buttner) [reply packages/endpoint/0.13.0/manifest.yml]: Yeah I think product wanted it to stay as `beta`.

File: packages/system/0.5.2/manifest.yml (hunk 1 above)
Comment (andresrc) [reply packages/system/0.5.2/manifest.yml:11]: Yes, for 7.10 we will need to release a new version

**Comment by ruflin** [comment]
Closing this PR as I think we should have a look at this again when newer versions of the packages are pushed and not do it now. Sorry for the noise.

**Pull request closed without merging**